src/services/todos.ts

```diff
@@ -0,0 +1,14 @@
+import todosFromServer from '../api/todos';
+import { Todo } from '../types/Todo';
+import { getUserById } from './user';
+
+export function getPreparedTodos() {
+  return todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }));
+}
+
+export function getMaxId(todos: Todo[]): number {
+  return Math.max(...todos.map(todo => todo.id), 0);
```

Comment:
The `getMaxId` function uses `Math.max` with a default value of 0, which is good for handling empty arrays. Ensure that todos always have valid IDs to avoid unexpected results.